tests/cargo-fmt/main.rs

```diff
@@ -0,0 +1,70 @@
+// Integration tests for cargo-fmt.
+
+use std::env;
+use std::process::Command;
+
+/// Run the cargo-fmt executable and return its output.
```

Comment:
Can you add comment that the tuple corresponds to `(stdout, stderr)` of the execution ?